pkg/kritis/admission/admission.go

```diff
@@ -152,6 +152,10 @@ func AdmissionReviewHandler(w http.ResponseWriter, r *http.Request) {
 }
 
 func reviewDeployment(deployment *appsv1.Deployment, ar *v1beta1.AdmissionReview) {
+	if checkBreakglassDeployment(deployment) {
+		glog.Infof("found breakglass annotation, returning successful status")
```

Comment:
This log message could be more useful if it mentioned the name of the deployment it found an annotation for.

Reply:
added names of deployment and pod for these error messages